src/pages/LogOutOldUserPage.js

```diff
@@ -0,0 +1,74 @@
+import React, {Component} from 'react';
```

Comment:
What does "old" reference in the name of this file?

Reply:
Oh, I think this means more like "previous" user. Let's change the name to "previous".

Reply:
Yeah `LogOutPreviousUserPage` is a much better name! I didn't mean to be agist 😂